### DISCUSSION TITLE
🧑🏼‍🏫🙋🙋‍♂️

### DISCUSSION THREAD

**Discussion by mmasias** [mmasias/24-25-EDA1#300] [General]
@Sergiorl2: porfa ve poniéndolos acá, usando su nombre de usuario de github.

**Comment by Sergiorl2** [comment]
_**(20/09/2024)**_
Maria Arroniz ([mariaarroniz](https://github.com/mariaarroniz))
Alejandro Juarez ([Alejandrojuarez0105](https://github.com/Alejandrojuarez0105))
Eirik Rosete ([eirik-rosete](https://github.com/eirik-rosete))

_**(27/09/2024)**_
Camila Lesly ([Camila-Lesly](https://github.com/Camila-Lesly))
Mario Viadero ([maariiooo](https://github.com/maariiooo))
Ian Saucedo ([Iansaucedo](https://github.com/Iansaucedo))
Miguel Castañeda ([micastg3](https://github.com/micastg3))

**_(14/10/2024)_**
Andrés López ([andreslb9](https://github.com/andreslb9))
Luca Recio ([Lukrecio](https://github.com/Lukrecio))
Ángel Efrain Romero ([aeRomeroz](https://github.com/aeRomeroz))
Bruno Maciejewski ([Bruno-MBl](https://github.com/Bruno-MBl))
Ian Saucedo ([Iansaucedo](https://github.com/Iansaucedo))

**_(08/11/2024)_**
Alejandro Viñas ([AlejandroVinas](https://github.com/AlejandroVinas))
Martín López ([martinlopez7](https://github.com/martinlopez7))
Ian Saucedo ([Iansaucedo](https://github.com/Iansaucedo))
Alejandro Juárez ([Alejandrojuarez0105](https://github.com/Alejandrojuarez0105))

**_(11/11/2024)_**
Ian Saucedo ([Iansaucedo](https://github.com/Iansaucedo))
Miguel Castañeda ([micastg3](https://github.com/micastg3))
Eirik Rosete ([eirik-rosete](https://github.com/eirik-rosete))
Celia Becerril ([celiabecerril](https://github.com/celiabecerril))
Kelvia Neves ([KelviaBarros05](https://github.com/KelviaBarros05))

**_(09/12/2024)_**
Alejandro Juarez ([Alejandrojuarez0105](https://github.com/Alejandrojuarez0105))
Marcos Gutiérrez ([[marcosgutierrez6](https://github.com/marcosgutierrez6)])
Ian Saucedo ([Iansaucedo](https://github.com/Iansaucedo))
